### PR TITLE
add hex verfmt to models

### PR DESCRIPTION
**Pull request by superm1**
I recall this was moved into the UX and database, so hopefully this was the only place needed.

LVFS side of https://github.com/fwupd/fwupd/pull/1842

### CODE REVIEW

File: lvfs/models.py [hunk 1]
```diff
@@ -965,6 +965,8 @@ def uint32_to_str(self, v):
             return '%i.%i.%i' % ((v & 0x00ff0000) >> 16,
                                  (v & 0x0000ff00) >> 8,
                                  v & 0x000000ff)
+        if self.value == 'hex':
+            return "%x" % v
```
Comment (hughsie) [inline lvfs/models.py]: Doesn't this need to be `%#x` to get the `0x` prefix?

**Comment by superm1** [comment]
Can you add to LVFS database too?

**Comment by hughsie** [comment]
> Can you add to LVFS database too?

Done.

**Comment by superm1** [comment]
You've got a typo.  Hexidemcial -> Hexadecimal

**Comment by hughsie** [comment]
It's Friday! It's close enough! Fixed :)

**Pull request merged**